README.md

```diff
@@ -56,15 +50,21 @@ snap install magma-access-gateway --classic
 magma-access-gateway.install
 ```
 
-> **NOTE:** To see the list of currently supported configuration options, execute:
+> **NOTE:** To see the list of currently supported installation options, execute:
 > ```bash
 > magma-access-gateway.install --help
 > ```
 
-After the system restarts, reconnect to the system and use `journalctl` to continue monitoring the 
-installation process. A successful installation will be indicated by the `Magma AGW deployment 
+> **NOTE:** If statically allocated IPs have been used, installer will perform automatic system 
+> restart to apply new network configuration. Once the server is restarted, reconnect to the system 
+> and use `journalctl` to continue monitoring the installation process.
+
+Successful installation will be indicated by the `Magma AGW deployment
```

Comment:
Does this message show up before the reboot? Because the note about the reboot comes after this one.

Reply:
Yes, it does.